src/components/ErrorMessage.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {isEmpty} from 'lodash';
+
+const ErrorMessage = ({ text, children, ...rest }) => {
+    if(isEmpty(text) && isEmpty(children)) {
+        return null
+    }
+
+   return  <div className="error" {...rest}>{text}</div>
```

Comment:
aren't you scared to destructure `rest` props directly to div. To native tags, we can pass only native props.